tests/erlang_tests/test_ets.erl

```diff
@@ -205,6 +231,17 @@ test_key_types_fun(Pid, _Options) ->
 
     Pid ! ok.
 
+get_port() ->
+    case erlang:system_info(machine) of
+        "ATOM" ->
+            open_port({spawn, "echo"}, []);
+        "BEAM" ->
+            erlang:binary_to_term(
```

Comment:
Beam is happy to spawn echo as
well on Unix, it just outputs some
garbage on stderr.

Reply:
fixed.